packages/css/src/components/row/row.scss

```diff
@@ -19,3 +19,47 @@
 .ams-row--wrap {
   flex-wrap: wrap;
 }
+
+.ams-row--align-around {
+  justify-content: space-around;
+}
+
+.ams-row--align-between {
+  justify-content: space-between;
+}
+
+.ams-row--align-center {
+  justify-content: center;
+}
+
+.ams-row--align-end {
+  justify-content: flex-end;
+}
+
+.ams-row--align-evenly {
+  justify-content: space-evenly;
+}
+
+.ams-row--align-start {
+  justify-content: flex-start;
+}
```

Comment:
This is default, do we need a class and option etc. for that?

Reply:
We don’t see a use case for it now, so we’ll be removing the classes setting defaults.